Sources/SwiftUICharts/SharedLineAndBar/Extras/LineAndBarEnums.swift

```diff
@@ -67,6 +67,9 @@ public enum DisplayValue {
     case yAxis(specifier: String, formatter: NumberFormatter? = nil)
     /// Places the label in the center of chart.
     case center(specifier: String, formatter: NumberFormatter? = nil)
+    /// Places the label at the opposite end of yAxis labels of the chart
+    case oppositeYAxis(specifier: String, formatter: NumberFormatter? = nil)
```

Comment:
Could it be `.position(at location: CGFloat, ...)` this place it as a percentile of total width within the chart?